crates/ruff_workspace/src/configuration.rs

```diff
@@ -59,64 +59,37 @@ pub struct RuleSelection {
 
 #[derive(Debug, Default)]
 pub struct Configuration {
```

Comment:
Please double check that I didn't remove any global option or forgot to move a linter specific option.

Reply:
Confirming that I spent some time looking through.